src/web/event-notifications/event-notification-details/SlackNotificationDetails.jsx

```diff
@@ -23,6 +23,7 @@ const SlackNotificationDetails = ({ notification }) => (
                            {notification.config.custom_message || <em>Empty body</em>}
                          </NewExampleWell>
                        )} />
+    <ReadOnlyFormGroup label="Message Backlog Limit" value={notification.config.backlog_size} />
```

Comment:
Should we be displaying something like `No limit` or `Disabled` when `backlog_size` is zero?

Reply:
sorry i donot follow your question .

Reply:
Non-issue.  Disregard.